types/jsesc/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for jsesc 0.4.3
+// Type definitions for jsesc 2.5.2
 // Project: https://github.com/mathiasbynens/jsesc
-// Definitions by: Bart van der Schoor <https://github.com/Bartvds>
+// Definitions by: Bart van der Schoor <https://github.com/Bartvds>, Lyanbin <https://github.com/Lyanbin>
```

Comment:
This should be one per line please - https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c5fb79f5359118ff6c44ee6267084d3e2478b004/types/alloy/index.d.ts#L3-L4

Otherwise looks good

Reply:
done.